tools/interface-mapping-validator/src/fixtures/legacy-sample.d.ts

```
@@ -0,0 +1,8 @@
+import { default as FRAuth } from './fake/fr-auth';
```

ESLint

[error] Missing file extension for "./fake/fr-auth"

(import/extensions)